Directory.Build.props

```
@@ -217,6 +218,7 @@
     <TargetstvOSSimulator Condition="'$(TargetOS)' == 'tvOSSimulator'">true</TargetstvOSSimulator>
     <TargetsAndroid Condition="'$(TargetOS)' == 'Android'">true</TargetsAndroid>
     <TargetsBrowser Condition="'$(TargetOS)' == 'Browser'">true</TargetsBrowser>
+    <TargetsWASI Condition="'$(TargetOS)' == 'wasi'">true</TargetsWASI>
```

We should probably change the casing here too:

```suggestion
    <TargetsWasi Condition="'$(TargetOS)' == 'wasi'">true</TargetsWasi>
```